sphinxcontrib/needs/utils.py

```diff
@@ -145,6 +156,22 @@ def rstjinja(app: Sphinx, docname: str, source):
     source[0] = rendered
 
 
+def import_prefix_link_edit(env, needs, id_prefix):
```

Comment:
instead of passing in the `env` object, are you able to just pass in the object you actually *need*? namely the `needs_extra_links`. as far as i can see that's the only `env` attribute you use in this method.

so something like

```python
def import_prefix_link_edit(needs: List[??], id_prefix: str, needs_extra_links: List[??])
   ...
```

Reply:
Changed it, I hope I got the type annotation correct.